proposals/lip-0061.md

```diff
@@ -570,7 +573,9 @@ def verifyAggregateCommit(blockHeader: BlockHeader)-> bool:
     # Note that the BFT parameters at height maxHeightCertified+1 are already authenticated
     # by the previous certificate.
     heightNextBFTParameters = getNextHeightBFTParameters(getBFTHeights().maxHeightCertified+1)
-    if previous function call returns valid height and aggregateCommit.height > heightNextBFTParameters-1:
+    if previous function call returns valid height
```

Comment:
Now that we explicitly use the return value, wouldn't it be better to write 

```suggestion
    if previous function call returns valid heightNextBFTParameters
```

or maybe even better use a try statement in the previous function call (catching the exception to return False)

Reply:
Good point, although it is just a cosmetic change, I think it makes it somewhat cleaner.

Reply:
Done, using the try except.

Reply:
I don't think that by catching the exception you return False here. I added try - except for the call of this function in `selectAggregateCommit` since it indeed looks cleaner.